.github/workflows/tagPublish.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           components: rustfmt, clippy
       - run: rustup set default-host ${{ matrix.platform.rust-target }}
       - name: install maturin
-        run: pip install 'maturin==0.8.3'
```

Comment:
Could you confirm maturin 0.9.0 allows `1-unchecked` option used `build_wheel.py`  file?  With the latest maturin version, I had obeserved that it does not work. 
Note: we cannot upload python bindings to pypi repo if manylinux option is turned off and we have used 1-unchecked since auth so libraries used **"libssl.so.1.1", "libcrypto.so.1.1"** is not manylinux compliant.

Reply:
Thanks Sandeep. You are right that we cannot upgrade to 0.9.0 as it does not support `1-unchecked`.